curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-100-arranged-probability.md

```diff
@@ -8,41 +8,80 @@ dashedName: problem-100-arranged-probability
 
 # --description--
 
-If a box contains twenty-one coloured discs, composed of fifteen blue discs and six red discs, and two discs were taken at random, it can be seen that the probability of taking two blue discs, P(BB) = (15/21)×(14/20) = 1/2.
+If a box contains twenty-one coloured discs, composed of fifteen blue discs and six red discs, and two discs were taken at random, it can be seen that the probability of taking two blue discs
```

Comment:
```suggestion
If a box contains twenty-one colored discs, composed of fifteen blue discs and six red discs, and two discs were taken at random, it can be seen that the probability of taking two blue discs.
```